lib/uo_cli.rb

```diff
@@ -68,7 +70,7 @@ class UOCLI
     # params\
     # +scenario+:: _string_ Name of csv file that defines the scenario\
     # +featureFile+:: _string_ Name of Feature File used to describe set of features in the district. If not passed, uses example project.
-    def scenario(scenario, featureFile=nil)
+    def run_func(scenario, featureFile=nil)
       featureFile = "example_project.json" if featureFile.nil?
```

Comment:
let's error out if featureFile is nil

Reply:
I removed the default featureFile from the function